### ISSUE TITLE
Cleaning up files

### ISSUE THREAD

**Issue by tlcmd** [geckolinux/geckolinux-project#106]
In trying to install Alien via the package manager, I screwed up and now have this page. How do I fix it?
Internal error. Please report a bug report with logs.
Run save_y2logs to get complete logs.

Caller: /usr/lib64/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'

Details: Failed to load Module 'Packages' due to: Failed to load Module 'SpaceCalculation' due to: cannot load such file -- storage

**Comment by tlcmd** [comment]
I finally gave up and re-installed GeckoLinux Mate. This is the 3rd time I re-installed GeckoLinux Mate for the same reason; I screwed up the package manager. 
My question: when it happens again, is there a way to fix the problems with package manager? I want to install Alien so I can convert .deb files to .rpm ones, so I suspect I'll mess up package manager again. 
Since I have both Windows 10 and Linux Mint  Mate 19, I can use one of them for this conversion until I find a way to fix a busted package manager. BTW, which of the offered Alien files should I try to download from GeckoLinux?

Thanks for any help and recommendations.
tlcmd
